daemons/gptp/common/ieee1588port.cpp

```diff
@@ -1274,17 +1276,20 @@ void IEEE1588Port::becomeSlave( bool restart_syntonization ) {
 
 	port_state = PTP_SLAVE;
 
-	/* TODO : Seems this be active here
+	/* TODO : Should this be active for non-automotive profile uses
```

Comment:
Not starting the sync receipt timer is the "right thing". I think there is (or at least was) a compliance test for this and there may be failures if we make a change and start the timer here. This is the correct interpretation of the state machine in the 802.1AS standard, but the state machine may not represent the correct or intended behavior.